extensions/wikia/WallNotifications/WallNotifications.class.php

```diff
@@ -387,50 +387,65 @@ protected function sendEmails( array $watchers, WallNotificationEntity $notifica
 			$watcher = $this->getUser( $val );
 			$mode = $watcher->getOption( 'enotifwallthread' );
 
-			if ( !empty( $mode ) && $watcher->getId() != 0 && (
+			if ( !( $watcher->getBoolOption('unsubscribed') === true ) &&
```

Comment:
Hmm, looks like there's going to be a lot of conflicts here actually...you guys were working on essentially the same part of code :)

I'd say we could definitely drop this line though, since we check if the user is unsubscribed in email extension.